apps/clinical/src/components/dashboardSection/styles/DashboardSection.module.scss

```
@@ -40,6 +40,17 @@ $table-cell-min-width: 6.875rem;
   }
 }
 
+.sectionNameWithEdit {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-block-size: fit-content !important;
+  p {
+    @include type-style("heading-compact-02");
+    color: $text-primary;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)